eng/cake/dotnet.cake

```diff
@@ -227,13 +227,17 @@ Task("dotnet-test")
             "**/Essentials.UnitTests.csproj",
             "**/Resizetizer.UnitTests.csproj",
             "**/Graphics.Tests.csproj",
-            "**/Compatibility.Core.UnitTests.csproj",
+            //"**/Compatibility.Core.UnitTests.csproj",
```

Comment:
Is this intentional? I see you are using this project in the if block below.

Reply:
Yeah didn't figure why these fail to run for now, we can enable after it's not important. But i will take a look again

Reply:
Ok I figured this one, the unit test project was missing from the .sln that the build cake target runs before. We should look at a better way for this, I wonder how this worked before. 

https://github.com/dotnet/maui/pull/21487/commits/a5af6c058463c72e4a2c99709ab5398afee71e88